src/java/org/apache/cassandra/db/compaction/CompactionTask.java

```diff
@@ -397,6 +397,7 @@ public static ImmutableCoordinatorLogOffsets getCoordinatorLogOffsets(Set<SSTabl
         ImmutableCoordinatorLogOffsets.Builder builder = new ImmutableCoordinatorLogOffsets.Builder();
         for (SSTableReader sstable : sstables)
             builder.addAll(sstable.getCoordinatorLogOffsets());
+        builder.purgeTransfers();
```

Comment:
wouldn't this cause us to promote sstables tables containing transfers that aren't fully reconciled?